src/ucs/sys/topo.h

```diff
@@ -33,6 +44,8 @@ typedef struct ucs_sys_device {
     int              numa_node; /**< NUMA node assoicated with the device*/
     ucs_sys_bus_id_t bus_id;    /**< bus ID of of the device if applicable.
                                    eg: 0000:06:00.0 {domain:bus:slot.function}*/
+    int              type_mask; /**< Describe if memory can be allocated on
```

Comment:
usually we use unsigned types for bitmasks

Reply:
I can change to unsgine but I noticed that there are bitmask variables in the code that are integer type as well.